CMakeLists.txt

```diff
@@ -110,6 +110,7 @@ endif()
 
```

Comment:
Here I need some more info on the purpose of this commit :-)

Reply:
Sorry, the GitHub webinterface wont tell me what you added this comment to, so I can't tell you.